lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install

```diff
@@ -118,6 +118,31 @@ function dosomething_rogue_schema() {
     ],
   ];
 
+  $schema['dosomething_rogue_signups'] = [
+    'description' => 'Table for tracking phoenix sid and corresponding rogue signups id and event id',
+    'fields' => [
+      'sid' => [
+        'description' => 'The sid of the signup as it is stored in phoenix.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'rogue_signup_id' => [
+        'description' => 'The signup id as it is stored in rogue.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'created_at' => [
+        'description' => 'The Unix timestamp of when the reportback was stored',
+        'type' => 'int',
```

Comment:
hmm maybe it should be but we don't have that elsewhere:
[reportbacks table](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install#L36-L38)
[failed table 1](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install#L97-L99)
[failed table 2](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install#L272-L274)